### PR TITLE
Create .readthedocs.yaml

### PR DESCRIPTION
**Pull request by tusharchou**
added read the docs

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```
Comment (mrutunjay-kinagi) [inline .readthedocs.yaml:12]: We have made it compatible for python version >=3.9

**Review by mrutunjay-kinagi — APPROVED** [review]
Approved !

**Comment by tusharchou** [comment]
#27 @mrutunjay-kinagi I ll need to change the structure to implement the suggestion 
Heads up changes coming store won't take iceberg class object in init it would go to target and source inits respectively . Are we aligned

**Comment by mrutunjay-kinagi** [comment]
Can you explain me more about this ?

**Pull request merged**